gaj23/fitlit-starter-kit
# Gaj/feature/html buildout

### What does this PR do? 
>Clearly and concisely describe the feature. 

This feature is the bare bone build out of the HTML. We have a summary section and three detailed sections. Each detailed section has containers that will later accept data. To my best of ability, ids and classes were included where they will be needed in future DOM manipulation. An assets folder was added with the files we will use. The images are from [flaticon](https://www.flaticon.com/).
<details>
<summary>Screen shots of HTML buildout</summary>
<img width="364" alt="Screen Shot 2020-12-08 at 4 41 52 PM" src="https://user-images.githubusercontent.com/68332132/101545243-fab3a980-3974-11eb-9555-711be0888f8b.png">
<img width="364" alt="Screen Shot 2020-12-08 at 4 42 07 PM" src="https://user-images.githubusercontent.com/68332132/101545244-fab3a980-3974-11eb-804d-3dde9cfc4e98.png">
</details>

### Is this a feature or refactor?  

This is a feature.

### Problems Encountered & Resulting Solutions  
>State any problems encountered 
>Describe your code changes in detail for reviewers.

None.

### Where should the reviewer start?
>What file(s) and line(s) are where changes are made?

Reviewers should look inside of src/index.html to review the written code. 

### How should this be tested?
>What steps need to be taken to confirm functionality? 

Reviewers should open src/index.html on their browser to compared to provided screenshots.

### Steps To Take
>Any questions and possible steps moving forward.

We will need to create the basic styling for our app.
Additionally, are the id and classes provided enough? Do we need more? Do we need less? And do they provide context for what elements they are related to?
